app/templating/summary/summary_item.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from app.templating.summary.formatter_registry import FormatterRegistry
+
+logger = logging.getLogger(__name__)
+
+
+class SummaryItem(object):
+    def __init__(self, schema, state_answers, question_type):
+        self.schema = schema
+        self.state_answers = state_answers
+        self.question_type = question_type
+        self.answer = self.format_answer()
+        self.question = schema.title or schema.answers[0].label
+        # Frontend url for each question
+        self.link = schema.container.container.id + '#' + schema.answers[0].id
+        self.type = self.schema.answers[0].type.lower()
+
+    # Formatting the answer for the frontend e.g. currency needs £
+    def format_answer(self):
+
+        user_answer = self.state_answers[0].value
+
+        if user_answer:
+            answer_type = self.schema.answers[0].type.upper()
+
+            # find out if the question type and answer has a formatter
+            formatter_class = FormatterRegistry.get_formatter(answer_type, self.question_type)
+            if formatter_class:
+                logger.debug("formatting answer %s", user_answer)
+                formatter_rule = formatter_class()
```

Comment:
Not sure where rule comes from, would be better to go with `formatter = formatter_class()`